Source/OIDIDToken.m

```diff
@@ -0,0 +1,152 @@
+/*! @file OIDIDToken.m
+    @brief AppAuth iOS SDK
+    @copyright
+        Copyright 2017 Google Inc. All Rights Reserved.
+    @copydetails
+        Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+ */
+
+#import "OIDIDToken.h"
+
+/*! Field keys associated with an ID Token. */
+static NSString *const kIssKey = @"iss";
```

Comment:
Any reason not to decode acr, amr, azp as well if present?

Reply:
None are used by the library directly, and everything is provided in the `claims` dictionary. I didn't decode all known types to reduce the scope. We *could* do it in the future, if there's value.

It's a bit like the other standard fields that we don't support of connect, and tell people just to use "additionalParams" for.